README.md

```diff
@@ -0,0 +1,313 @@
+
+### App Screenshot
+
+![App](https://github.com/peterdtitan/BnB-homes/assets/92449229/3f50b422-a9d3-4ece-92fb-6b96543dd4a4)
+
+
+### Initial State of Kanban Board
+
+![kanban](https://user-images.githubusercontent.com/92449229/256266758-11cd96b1-b12c-4d0e-ab5d-d4fba7afd40f.png)
+
+### Setup
+
+- Clone this repository to your desired folder:
+
+```sh
+   git clone https://github.com/peterdtitan/BnB-homes.git
+```
+
+- Navigate into the cloned folder
+
+```sh
+cd server
+```
+
+### Install
+
+Install this project with the following:
+
+```sh
+  bundle install
+```
+
+- Set up the database and change the username and password of your Postgres account in
+
+```sh
+config/database.yml
+```
+
+```sh
+rails db:create db:migrate db:seed
+```
+
+- Start the server
+
+```sh
+rails s
+```
+
+- Run the tests
+
+```sh
+rspec ./spec/models
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### :notebook: API Documentation <a name="api-docs"></a>
+
+- Run the server with `rails s` command
+- The open [API Documentation](https://localhost:3000/api-docs) in your browser.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+# 📖 [BnB-homes Front-End](https://github.com/peterdtitan/BnB-homes/tree/dev/server) <a name="about-project"></a> <a name="about-project"></a>
+
+<b>BnB-homes</b> is a website where users can book a homes, cancel a booking. The user can also see the details of a homes.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li>Redux</li>
+    <li>CSS</li>
+  </ul>
+</details>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- [x] A web browser like Google Chrome.
+- [x] A code editor like Visual Studio Code with Git and Node.js.
+
+You can check if Git is installed by running the following command in the terminal.
+```
+$ git --version
+```
+
+Likewise for Node.js and npm for package installation.
+```
+$ node --version && npm --version
+```
+### Setup
+
+Clone this repository using the GitHub link provided below.
+
+
+### Install
+
+In the terminal, go to your file directory and run this command.
+
+```
+$ git clone https://github.com/peterdtitan/BnB-homes.git
+```
+
+- Navigate into the cloned folder
+
+```sh
+cd client
+```
+
+```bash
+
+$ npm install
+```
+
+### Usage
+
+
+Kindly modify the files as needed.
+
+In the project directory, you can run:
+```
+$ npm start
+```
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.
+
+You may also see any lint errors in the console.
+
+
+
+### Run tests
+
+To run tests, run the following command:
+
+- To Test:-
+
+```bash
+   npm test
+```
```

Comment:
- [ ] After the recent changes, some frontend tests are failing:

![image](https://github.com/peterdtitan/BnB-homes/assets/57421823/c6dbe8eb-1921-4269-8946-489206989aa9)

![image](https://github.com/peterdtitan/BnB-homes/assets/57421823/730432e3-e453-4256-ade9-66e16be857fd)

In the last review, these tests were passing.  

Kindly, take a look at those tests so they pass again. 👍